README.md

```diff
@@ -56,8 +56,19 @@ var NukiBridgeApi = require('nuki-bridge-api');
 var lockStates = NukiBridgeApi.lockState;
 var lockActions = NukiBridgeApi.lockAction;
 
-...
+var bridge = new NukiBridgeApi.Bridge(ip, port, token);
 
+bridge.list().then(function gotNukis (nukis) {
+    nukis.forEach(function(nuki) {
```

Comment:
I'm sorry I'm not too good with the functionality of Promises, so you have to help me with this. My understanding was, that ` bridge.list().then(function gotNukis (nukis) { ` will return all Nuki's available (and thus wait for all of them to be returned), so that `     nukis.forEach(function(nuki) { ` will always loop through them all?
Do I have to change my code to ` bridge.list().map(function(nuki) { ... }) ` in order to always go through all Nukis?

Reply:
The problem here would be that the forEach over the nukis is not a good idea, because calls on one nuki cant be awaited for.

I just found out, that `bridge.list().map()` would not work, as `request-promise` only returns the real bluebird-promise when calling `.promise()` on it.

So maybe we can express it like this:
```js
var bridge = new NukiBridgeApi.Bridge(ip, port, token);

var nukis = bridge.list().promise(); // Bluebird Promise

nukis.map(function closeAll (nuki) {
   return nuki.lockAction(lockAction.LOCK);
}).then(function afterCloseAll () {
   console.log('All doors are closed now!');
});
```

This is the correct way to wait for something called on all nukis.

Reply:
Sorry for the delay. I only get to develop on the weekends now.
I will update the documentation based on our discussion.

I was wondering though, where I could use `catch()` in case the connection settings are wrong?
My code before our discussions:

```
bridge.list().then(function gotNukis(nukis)
{
	nukis.forEach(function(nuki)
	{
		...
	});
})
.catch(function(e)
{
	adapter.log.warn('Connection settings for bridge incorret' + (bridge.data.bridge_name ? ' with name ' + bridge.data.bridge_name : (bridge.data.bridge_id ? ' with ID ' + bridge.data.bridge_id : (bridge.data.bridge_ip ? ' with ip ' + bridge.data.bridge_ip : ''))) + '! No connection established.');
	adapter.log.debug(e.message);
});
```

Reply:
Catch will only catch a rejected list, not the things that happen in the synchronous forEach. 
And `adapter.log` is confusing too, since `adapter` has nothing to do with this project.

Anyway I think that correct usage of promises should be part of learning NodeJS or Javascript in general, so eventually I think this doc is not needed.
I think I will close this without changing the existing code/doc.

Reply:
`adapter` was for my to debug the code in my dev environment. Is of course nothing for the documentation here.
I agree though regarding the doc